.github/workflows/rust.yml

```diff
@@ -60,22 +60,33 @@ jobs:
     - name: Set environment variables
       run: |
         echo "LINERA_STORAGE_SERVICE=127.0.0.1:1235" >> "$GITHUB_ENV"
-
+        echo "LINERA_WALLET=/tmp/WORK/wallet_0.json" >> "$GITHUB_ENV"
+        echo "LINERA_STORAGE=rocksdb:/tmp/WORK/client_0.db" >> "$GITHUB_ENV"
+        echo "LINERA_FAUCET_URL=http://localhost:8079" >> "$GITHUB_ENV"
     - name: Build example applications
       run: |
         cd examples
         cargo build --locked --release --target wasm32-unknown-unknown
-    - name: Run end-to-end tests
+    - name: Run the storage-service instance and the storage-service tests
       run: |
         cargo run --release -p linera-storage-service -- memory --endpoint $LINERA_STORAGE_SERVICE &
         cargo test --features storage-service,unstable-oracles -- storage_service --nocapture
     - name: Run Ethereum tests
       run: |
         cargo test -p linera-ethereum --features ethereum
         cargo test test_wasm_end_to_end_ethereum_tracker --features ethereum,storage-service,unstable-oracles
+    - name: Run the validators
+      run: |
+        cargo build --features storage-service
+        mkdir /tmp/WORK
+        cargo run --features storage-service --bin linera -- net up --storage-config-namespace service:tcp:localhost:1235:table --policy-config devnet --path /tmp/WORK &
     - name: Compile Wasm test modules for Witty integration tests
       run: |
         cargo build -p linera-witty-test-modules --target wasm32-unknown-unknown
+    - name: Run the remote-net tests
+      run: |
+        cargo run --bin linera -- faucet --amount 1000 --port 8079 &
+        cargo test -p linera-service remote_net_grpc --features remote-net
```

Comment:
We should probably run this with the `unstable-oracles` feature.

Reply:
No, because this creates the error ```
error: none of the selected packages contains these features: unstable-oracles
```